frontend/cypress/e2e/beta/make-and-review-access-request.cy.ts

```diff
@@ -0,0 +1,62 @@
+const baseURL = '/beta'
+let modelUuid = ''
+let accessRequestUuid = ''
+const modelName = 'Test Model'
+const schemaId = 'minimal-access-request-general-v10-beta'
+
+describe('Make and approve an access request', () => {
+  before(() => {
+    cy.log('Upload new model and set schema via API')
+    cy.request('POST', 'http://localhost:8080/api/v2/models', {
+      name: modelName,
+      teamId: 'Uncategorised',
+      description: 'This is a test',
+      visibility: 'public',
+    }).then((response) => {
+      expect(response.status).to.eq(200)
+      expect(response.body.model).to.have.property('name', modelName)
+      modelUuid = response.body.model.id
+      cy.request('POST', `/api/v2/model/${modelUuid}/setup/from-schema`, { schemaId: 'minimal-general-v10-beta' }).then(
+        (response) => {
+          expect(response.status).to.eq(200)
+          expect(response.body.card).to.have.property('modelId', modelUuid)
+        },
+      )
+    })
+  })
+
+  it('can make an access request', () => {
+    cy.visit(`${baseURL}/model/${modelUuid}`)
+    cy.contains(modelName)
+    cy.get('[data-test=accessRequestTab]').click()
+    cy.get('[data-test=accessRequestButton]').click()
+    cy.url({ timeout: 15000 }).should('contain', `/beta/model/${modelUuid}/access-request/schema`)
```

Comment:
Why has the timeout been increased here? 10 seconds should be more than enough time for the URL to update